src/addMessage/addMessage.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import './addMessage.css'
+import { useDispatch, useSelector } from 'react-redux';
+import { addComment } from '@/features/allComments/allComments';
+import { v4 } from 'uuid';
+import moment from 'moment/moment';
+import { allAccountsList } from '@/store/selectors';
+import { changeCurrentUser } from '@/features/currentUser/currentUser';
+const AddMessage = () => {
+    const allAccounts = useSelector(allAccountsList)
+    const [currentTextAreaValue, setCurrentTextAreaValue] = useState('')
+    const [currentUser, setCurrentUser] = useState(allAccounts[0])
+    const dispatch = useDispatch()
+    const addMessageHandler = (event) => {
+        event.preventDefault();
+        const messagetInf = {
+            postId: v4(),
+            userId: currentUser.id,
+            text: currentTextAreaValue,
+            userName: currentUser.accountName,
+            date: moment().format('YYYY-MM-DD HH:mm'),
+            pathToProfilePhoto: currentUser.pathToPhoto,
```

Comment:
so you copy data from `currentUser` here. but what you can do is just save the id of the user who performed the operation. 
```ts
const message = {
  authorId: currentUser.id
}
```
and then you can find the corresponding user at render time and render their data.
you can get inspiration from [DB normalization](https://en.wikipedia.org/wiki/Database_normalization), a lot of its concepts can be applied to redux store. definitely there are cases when you'll need to copy data from one object to the other, but here it's better to just save a reference to the other object